app/src/main/java/blockeq/com/stellarwallet/utils/StringFormat.kt

```diff
@@ -17,5 +17,26 @@ class StringFormat {
             if (string == null) return ""
             return String.format("%.4f", string.toDouble())
         }
+
+        /**
+         * Calculate the number of decimals of a string.
+         */
+        fun getNumDecimals(num: String): Int {
+            return if (!hasDecimalPoint(num)) {
+                0
+            } else num.substring(num.indexOf('.') + 1, num.length).length
```

Comment:
The formatting of the code here makes it confusing to read - if we have curly brackets on if, then it's expected else statement will also have them.